test/Groups/pcgroup.jl

```diff
@@ -82,3 +82,13 @@ end
   @test GAP.Globals.IsMutable(cgg)
   @test cgg !== c.X
 end
+
+@testset "create collectors from polycyclic groups" begin
+  for i in rand(1:number_small_groups(96), 10)
```

Comment:
```suggestion
  for i in rand(1:number_of_small_groups(96), 10)
```
the old version is deprecated

Reply:
o.k.